### ISSUE TITLE
NEED HELP: Action Classification low performance

### ISSUE THREAD

**Issue by Jmei1994** [OpenGVLab/InternVideo#127]
Problems: Use demo to test action classification on kinetics-700 validation set but get very poor result

Experiment: 
1. Pretrained model: https://huggingface.co/OpenGVLab/InternVideo2-Stage2_1B-224p-f4/tree/main
2. text candidate：use the class name of k700 dataset annotation
3. dataset：kinetics-700 validation set.
4. code：demo.ipynb

example：
input：carving ice/nTnAoTQ41Nc_000011_000021.mp4
output：
text: coloring in ~ prob: 0.0085
text: acting in play ~ prob: 0.0053
text: smashing ~ prob: 0.0043
text: cracking knuckles ~ prob: 0.0041
text: tasting food ~ prob: 0.0040

**Comment by Andy1621** [comment]
Can you try the VideoCLIP model?

**Comment by JayMay1994** [comment]
> Can you try the VideoCLIP model?

Do you mean InternVideo2_CLIP in InternVideo2/multi_modality/scripts/evaluation/clip/zero_shot/1B/config_k400.py  ？

**Comment by JayMay1994** [comment]
> Can you try the VideoCLIP model?

Do you mean abandon  internvideo2 and turn to videoclip for action recognition？Your work seems very promising.

**Comment by Andy1621** [comment]
Yes. I'm not sure whether there is a bug in the demo. But I have tested the [VideoCLIP](https://github.com/OpenGVLab/InternVideo/blob/main/InternVideo2/multi_modality/scripts/evaluation/clip/zero_shot/1B/eval_k700.sh) on Kinetics and it runs normally.
